### PR TITLE
Dev apfixes

### PR DESCRIPTION
**Pull request by AndrePaulGrenon**
Big cleanup, removing console log, fixing minor stuff and adding comments. Login42 ImgUpload frontend, also backen token services auth and imgtransfer. Lot cleaner.

### CODE REVIEW

**Comment by AndrePaulGrenon** [comment]
This is just a major cleanup and some minor fixes so the auth and tokenverification is cleaner and easier to read.

File: app/backend/src/auth/auth.service.ts
Comment (chughes741) [inline app/backend/src/auth/auth.service.ts:1]: Seems to be a lot of random whitespace added. Was Prettier run before commiting?

**Review by chughes741 — CHANGES_REQUESTED** [review]
Run yarn format

File: app/backend/src/auth/auth.service.ts
Comment (AndrePaulGrenon) [reply app/backend/src/auth/auth.service.ts:1]: Done

**Comment by AndrePaulGrenon** [comment]
Yarn Format push

**Review by chughes741 — APPROVED** [review]

**Pull request merged**